code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -159,10 +159,10 @@
 
 /// Open the cell cover when ALT+Click on the suit
 /obj/item/clothing/suit/space/AltClick(mob/living/user)
-	. = ..()
-	if(!istype(user) || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
+	if(istype(user) && user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
+		toggle_spacesuit_cell(user)

```

Comment:
revert all this, `return ..()` if the if passes instead

Reply:
can remove the istype too tbh

Reply:
removed